### PR TITLE
Add the NYCharities to the map with details upon highlight.

### PR DESCRIPTION
**Pull request by joeleu**
This PR adds the NY Charities data to the map collapsed into an icon by default. Upon hovering over an icon some of the basic details for the organization are presented to the.

Example screenshot:

Screenshot 2023-04-01 at 21 12 41

### CODE REVIEW

File: voluncheer/static/css/map.css [hunk 1]
```diff
@@ -40,5 +44,163 @@ body {
 }
 
 .text-center {
-    text-align: center!important;
+    text-align: center !important;
+}
+
+.org {
+    background-color: #4285F4;
```
Comment (Hgersten-hash) [inline voluncheer/static/css/map.css:51]: Can we add a border around the markers so that it is easier to differentiate when they are close together?

**Review by Hgersten-hash — CHANGES_REQUESTED** [review]
LGTM! Can we just add a border around the markers so that it is easier to differentiate when they are close together?

**Review by JinweiZhu — APPROVED** [review]
Good job.

**Review by EdenWuyifan — APPROVED** [review]
LGTM

**Pull request merged**